### DISCUSSION TITLE
C#

### DISCUSSION THREAD

**Discussion by mattmazer0397** [WhitewaterFoundry/Pengwin#666] [Q&A]
I am about officially re-enter college and I think I'll decide on learning C# (my school offers both C# and Java, and I'm a bit familier with both). I was wondering if anyone in the Pengwin community can share their experiences in coding with C#, and just an honest review of using Pengwin and something like VSCode over Win10 and Visual Studio.

Also, I assume the .NET package in pengwin-setup contains everything I need to get started?

**Comment by crramirez** [comment]
Hello @mattmazer0397 ,

The dotnet installer in pengwin-setup will install the .NET 5 SDK. It is everything that you need. You can also just install Visual Studio and learn .NET in the Windows way.

I will let others talk about their experience using Visual Studio Code. As I understand Visual Studio Code .NET integration with WSL will use mono, but things may have changed since I tested it.

Regards,
Carlos
